src/sdk/namespace_fs.js

```diff
@@ -1419,23 +1419,22 @@ class NamespaceFS {
                 const versioned_path = latest_ver_info && this._get_version_path(key, latest_ver_info.version_id_str);
                 const versioned_info = latest_ver_info && await this._get_version_info(fs_context, versioned_path);
 
-                gpfs_options = is_gpfs ?
-                    await this._open_files_gpfs(fs_context, new_ver_tmp_path, latest_ver_path, upload_file,
-                        latest_ver_info, open_mode, undefined, versioned_info) :
-                    undefined;
                 const bucket_tmp_dir_path = this.get_bucket_tmpdir_full_path();
-                dbg.log1('Namespace_fs._move_to_dest_version:', latest_ver_info, new_ver_info, gpfs_options);
-
-                if (this._is_versioning_suspended()) {
+                if (!is_gpfs && this._is_versioning_suspended()) {
```

Comment:
can you just add a comment, something like - 
on GPFS & while versioing is suspended & latest version is a null version, we can skip unlinking the old version because linkatif overrides.